pkg/sshd/lock.go

```diff
@@ -17,7 +17,7 @@ type RepositoryLock interface {
 	// Lock acquires a lock for a repository. In the case the repository is already locked
 	// it waits until a timeout to get the lock. If it was not possible to get the
 	// lock after the timeout an error is returned.
-	Lock(repoName string, timeout time.Duration) error
```

Comment:
@smothiki for some background, @aledbf originally added this timeout so that we could build distributed locks with timeouts. since we've decided not to support multiple builder instances for now, we don't need that functionality right now.

However, if you are removing this, can you remove the godoc above that references it?